pkg/controller/master/rancher/embedded_rancher.go

```diff
@@ -4,17 +4,8 @@ import (
 	"strings"
 
 	rancherv3api "github.com/rancher/rancher/pkg/apis/management.cattle.io/v3"
-	"github.com/sirupsen/logrus"
-
-	"github.com/harvester/harvester/pkg/settings"
 )
 
-var UpdateRancherUISettings = map[string]string{
-	"ui-dashboard-index":   settings.DefaultDashboardUIURL,
-	"ui-offline-preferred": "false",
-	"ui-pl":                "Harvester",
```

Comment:
the `ui-pl: Harvester` is required since the UI uses this to identify whether it is a Harvester cluster before login.

Reply:
Updated